example/example-tzip16.ts

```diff
@@ -4,7 +4,7 @@ import { tzip16, Tzip16Module } from '@taquito/tzip16';
 async function example() {
   try {
 
-    const tezos = new TezosToolkit('https://api.tez.ie/rpc/florencenet');
+    const tezos = new TezosToolkit('https://api.tez.ie/rpc/granadanet');
```

Comment:
The example won't work because KT1T2KNHdrPwLE4TZcGr6mVZt1s6k9j7tHCV is on florencenet

Reply:
I will fix example cases in next PR